.github/pull_request_template.md

```diff
@@ -18,10 +18,9 @@ Tests <!-- At least one of them must be included. -->
 
 Code changes
 
- - Has Helm charts change
  - Has Go code change
  - Has CI related scripts change
- - Has documents change
```

Comment:
We tend to deprecate helm and documents are now maintained in the dedicated repo